src/components/layout/Booking/BookingForm/BookingFormContent/BookingFormContent.jsx

```diff
@@ -166,25 +164,20 @@ class BookingFormContent extends PureComponent {
           </p>
         )}
 
-        {!isEvent && canExpire && isDigital && !autoActivateDigitalBookings && (
+        {!isEvent && canExpire && isDigital && !hasActivationCode && (
           <p className="bc-notification">
             {
               'Tu as 30 jours pour faire valider ta contremarque. Passé ce délai, ta réservation sera automatiquement annulée.'
             }
           </p>
         )}
 
-        {!isEvent &&
-          canExpire &&
-          isDigital &&
-          hasActivationCode &&
-          autoActivateDigitalBookings &&
-          enableActivationCodes && (
-            <p className="bc-notification">
-              {
-                "Pour cette offre numérique, ta réservation sera définitivement validée. Tu ne pourras pas l'annuler par la suite."
-              }
-            </p>
+        {!isEvent && canExpire && isDigital && hasActivationCode && (
+          <p className="bc-notification">
+            {
+              "Pour cette offre numérique, ta réservation sera définitivement validée. Tu ne pourras pas l'annuler par la suite."
+            }
+          </p>
```

Comment:
Tout ce bout de code est assez difficile à lire, j'aurai bien bougé la logique dans une fonction du composants comme :

```suggestion
getBookingValidationNotification = () => {
  const {
    isDigital, 
    hasActivationCode 
  } = this.props

  if (isDigital) {
    if (hasActivationCode) {
      return "Pour cette offre numérique, ta réservation sera définitivement validée. Tu ne pourras pas l'annuler par la suite."
    } else {
      return 'Tu as 30 jours pour faire valider ta contremarque. Passé ce délai, ta réservation sera automatiquement annulée.'
    }
  }

  return 'Tu as 30 jours pour récupérer ton bien et faire valider ta contremarque. Passé ce délai, ta réservation sera automatiquement annulée.'
}
// ...
{ !isEvent && canExpire && (
  <p className="bc-notification">
    {getBookingValidationNotification()}
  </p>
)}

```